GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -2733,16 +2735,15 @@ private void DiffContextMenu_Opening(object sender, System.ComponentModel.Cancel
                 artificialRevSelected = artificialRevSelected || selectedRevisions[selectedRevisions.Count - 1].IsArtificial();
 
             // disable items that need exactly one selected item
-            bool isExcactlyOneItemSelected = DiffFiles.SelectedItems.Count() == 1;
-            var isCombinedDiff = isExcactlyOneItemSelected &&
+            bool isExactlyOneItemSelected = DiffFiles.SelectedItems.Count() == 1;
+            var isCombinedDiff = isExactlyOneItemSelected &&
                 DiffFiles.CombinedDiff.Text == DiffFiles.SelectedItemParent;
-            var enabled = isExcactlyOneItemSelected && !isCombinedDiff;
```

Comment:
Hmm, to be honest, I didn't (and still don't) understand that this meant a combined diff **of a merge commit**:

```
var isCombinedDiff = isExactlyOneItemSelected &&
                  DiffFiles.CombinedDiff.Text == DiffFiles.SelectedItemParent;
```

I suppose, then, that I'd want to do:

```
openWithDifftoolToolStripMenuItem.Enabled = !isCombinedDiff;
```

Does that seem sensible? I'll test it out as soon as I can.